cips/cip-shwap_protocol.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Shwap Protocol 
+description: Shwap - the new extensible sampling protocol 
+author: Hlib Kanunnikov (@Wondertan)
+discussions-to: https://forum.celestia.org/t/cip-shwap-protocol/1551
+status: Draft
+type: Standards Track
+category: Data Availability, Networking
+created: 2024-02-02
+---
+
+## Abstract
+
+This document specifies Shwap - the simple and expressive yet extensible and future-proof messaging framework aiming to
```

Comment:
just started review. I would recommend if we use key words like "future-proof" to also expand on why it is future-proof later in the document. Haven't fully read doc but wanted to flag here in case.

Reply:
i see no mention of "future-proof" in the doc now, so marking this resolved